
From: NeilBrown <neilb@cse.unsw.edu.au>

NFSv4 should really run over TCP, and clients will expect that; so there's no
point letting people build kernels that support NFSv4 without also supporting
server-side TCP.

Signed-off-by: J. Bruce Fields <bfields@citi.umich.edu>
Signed-off-by: Neil Brown <neilb@cse.unsw.edu.au>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/Kconfig |    1 +
 1 files changed, 1 insertion(+)

diff -puN fs/Kconfig~nfsd-force-server-side-tcp-when-nfsv4-enabled fs/Kconfig
--- 25/fs/Kconfig~nfsd-force-server-side-tcp-when-nfsv4-enabled	2004-08-09 22:01:57.721975360 -0700
+++ 25-akpm/fs/Kconfig	2004-08-09 22:01:57.726974600 -0700
@@ -1495,6 +1495,7 @@ config NFSD_V3
 config NFSD_V4
 	bool "Provide NFSv4 server support (EXPERIMENTAL)"
 	depends on NFSD_V3 && EXPERIMENTAL
+	select NFSD_TCP
 	help
 	  If you would like to include the NFSv4 server as well as the NFSv2
 	  and NFSv3 servers, say Y here.  This feature is experimental, and
_
